packages/EasyErrorHandler/src/Providers/ErrorCodesByInterfaceProvider.php

```diff
@@ -9,7 +9,7 @@
 use ReflectionException;
 use Symfony\Component\ErrorHandler\Error\ClassNotFoundError;
 
-final class ErrorCodesProvider implements ErrorCodesProviderInterface
+final class ErrorCodesByInterfaceProvider implements ErrorCodesProviderInterface
```

Comment:
What about `ErrorCodesByInterfaceProvider` -> `ErrorCodesFromInterfaceProvider` and `ErrorCodesFromEnumsProvider` respectively? And the same for all aliases in configuration.

Reply:
Renamed